### PR TITLE
Add SMTP env var templating

### PR DESCRIPTION
**Pull request by deer-wmde**
https://phabricator.wikimedia.org/T295142

### CODE REVIEW

**Comment by addshore** [comment]
Needs https://github.com/wbstack/mediawiki/pull/152

File: charts/mediawiki/templates/_helpers.tpl [hunk 1]
```diff
@@ -112,6 +118,22 @@ Common deployment environment variables
 - name: MW_LOG_TO_STDERR
   value: "yes"
 {{- end }}
+- name: MW_SMTP_ENABLED
+{{- if .Values.mw.smtp.enabled }}
+  value: "yes"
+{{- else }}
+  value: "no"
+{{- end }}
+- name: MW_SMTP_HOST
```
Comment (addshore) [inline charts/mediawiki/templates/_helpers.tpl:127]: I wonder if these env var should also be wrapped up in a conditional some how?
But I don't think' it matters too much / we don't have a best practice right now

**Comment by addshore** [comment]
I tagged `1.36-7.4-20211108-0` of the mediawiki repo, which will be needed by this chart release.

File: charts/mediawiki/templates/_helpers.tpl (hunk 1 above)
Comment (deer-wmde) [reply charts/mediawiki/templates/_helpers.tpl:127]: Good question - it probably wouldnt change anything as far as I can see but it could be cleaner?

**Pull request merged**